### DISCUSSION TITLE
Database Schemas

### DISCUSSION THREAD

**Discussion by Dharmik48** [TDEC-surat/TDEC-Register#25] [Q&A] [answered]
How many Models will there be in the DB (like user, leave-application, etc.) and what data will it hold (like user will hold username, password, their leave applications, etc.) ?

**Comment by Dharmik48** [comment, marked as answer]
Please respond as fast as possible

**Reply by AkshDesai04** [reply]
Everything in the registration page plus...

ID
Password
Total leaves taken so far (CL, ML, Off and so on separately)
A column for approved ones from sir so as to manage approved and applied leaves

**Reply by AkshDesai04** [reply]
(Not sure about the last one

We may use a diff methods to store approved ones)

**Reply by Dharmik48** [reply]
let's start off simple and start with including only the basic data like _id_ and _password_

**Reply by AkshDesai04** [reply]
Yep sounds cool👍

(The password is to be a 4-6 digit integer only FYI)

**Reply by Dharmik48** [reply]
Why only Integer?

**Reply by AkshDesai04** [reply]
The client asked for it

I was like ya sure🤷. I mean it's your project;u decide🤷

😂

**Reply by Dharmik48** [reply]
ohk then👍
